package.json

```
@@ -4,11 +4,12 @@
   "private": true,
   "source": "src/index.html",
   "scripts": {
+    "mock": "copyfiles -u 1 static/mockServiceWorker.js dist",
     "clean": "rimraf ./dist && rimraf ./.parcel-cache",
-    "dev": "yarn run clean && parcel",
-    "build": "yarn run clean && parcel build",
+    "dev": "yarn run clean && npm run mock && parcel",
```

Can we handle all the scripts with `yarn`?

```suggestion
    "dev": "yarn run clean && yarn run mock && parcel",
```

---

To be consistent, we should also delete the package-lock in favor of yarn-lock